### ISSUE TITLE
[Accessibility: ACC-270.1, ACC-270.6] Button: UI cannot be activated and is not available once search value is entered.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12582]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
It must be possible to invoke/activate each & every interactive UI, using the keyboard.
Keyboard is the prerequisite for support of assistive technologies to enable users with disabilities to read and operate software applications, hence all the UIs should be activated with keyboard.
Each & every UI, must be accessible using the keyboard and should not disappear once search value is entered.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [VHD](https://sap.github.io/fundamental-ngx/#/platform/value-help-dialog)
‘Synchronize’ button within VHD cannot be activated using keyboard, mouse and touch. ‘Synchronize’ button in not available once search value is entered.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/40a6df51-48a6-4264-aebe-9b00fb8c20a5">
<img width="452" alt="image" src="https://github.com/user-attachments/assets/32079a18-1992-425f-b7c6-4ef9933d2c24">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
I am not able to reproduce the issue. The button is accessible with tabing
![VHD](https://github.com/user-attachments/assets/84e5da39-ae8e-4d00-8f03-671089dab18d)
